### PR TITLE
Add info on using the OSG 23 tarballs (SOFTWARE-5701)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by timtheisen** [comment]
I think that Brian wanted to use 23-main for the version instead of 23.

File: docs/worker-node/install-wn-tarball.md [hunk 1]
```diff
@@ -27,10 +27,16 @@ Download the WN Client
 
 Please pick the `osg-wn-client` tarball that is appropriate for your distribution and architecture. You will find them in <https://repo.opensciencegrid.org/tarball-install/> .
 
+For OSG 23:
+
+-   [Binaries for RHEL8-compatible](https://repo.opensciencegrid.org/tarball-install/23/osg-wn-client-latest.el8.x86_64.tar.gz)
+-   [Binaries for RHEL9-compatible](https://repo.opensciencegrid.org/tarball-install/23/osg-wn-client-latest.el9.x86_64.tar.gz)
```
Comment (brianhlin) [inline docs/worker-node/install-wn-tarball.md:33]:
```suggestion
-   [Binaries for RHEL8-compatible](https://repo.opensciencegrid.org/tarball-install/23-main/osg-wn-client-latest.el8.x86_64.tar.gz)
-   [Binaries for RHEL9-compatible](https://repo.opensciencegrid.org/tarball-install/23-main/osg-wn-client-latest.el9.x86_64.tar.gz)
```

To mirror what we have in https://repo.opensciencegrid.org/osg/

File: docs/worker-node/install-wn-tarball.md [hunk 2]
```diff
@@ -49,13 +55,13 @@ Install the WN Client
...
 
-Example installation (in `/home/user/test-install`, the **`<PATH_TO_CLIENT>/`** is `/home/user/test-install/osg-wn-client` ):
+Example EL9 installation (in `/home/user/test-install`, the **`<PATH_TO_CLIENT>/`** is `/home/user/test-install/osg-wn-client` ):
 
 ```console
 user@host $ mkdir /home/user/test-install
 user@host $ cd /home/user/test-install
-user@host $ wget https://repo.opensciencegrid.org/tarball-install/3.6/osg-wn-client-latest.el7.x86_64.tar.gz
-user@host $ tar xzf osg-wn-client-latest.el7.x86_64.tar.gz
+user@host $ wget https://repo.opensciencegrid.org/tarball-install/23/osg-wn-client-latest.el9.x86_64.tar.gz
```
Comment (timtheisen) [inline docs/worker-node/install-wn-tarball.md:63]:
```suggestion
user@host $ wget https://repo.opensciencegrid.org/tarball-install/23-main/osg-wn-client-latest.el9.x86_64.tar.gz
```

**Comment by matyasselmeci** [comment]
I think I fixed the issues; anything else?

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**